public/js/passwordindicator.js

```diff
@@ -0,0 +1,59 @@
+const indicator = document.querySelector(".indicator");
+const input = document.querySelector(".password");
+const weak = document.querySelector(".weak");
+const medium = document.querySelector(".medium");
+const strong = document.querySelector(".strong");
+const text = document.querySelector(".text");
+const showBtn = document.querySelector(".showBtn");
+let regExpWeak = /[a-z]/;
+let regExpMedium = /\d+/;
+let regExpStrong = /.[!,@,#,$,%,^,&,*,?,_,~,-,(,)]/;
+function trigger(){
+  if(input.value != ""){
+    indicator.style.display = "block";
+    indicator.style.display = "flex";
+    if(input.value.length <= 3 && (input.value.match(regExpWeak) || input.value.match(regExpMedium) || input.value.match(regExpStrong)))no=1;
```

Comment:
Make this code a bit better readable, like you can do one thing.
If you are using VScode, simply select the whole filea and right click and format it.

It would be better with some spaces and line breaks